aioshelly/wsrpc.py

```diff
@@ -202,10 +210,16 @@ def connected(self) -> bool:
         """Return if we're currently connected."""
         return self._client is not None and not self._client.closed
 
-    async def call(self, method, params=None, timeout=10):
+    async def call(
+        self, method: str, params: dict[str, Any] | None = None, timeout: int = 10
+    ) -> Any:
         """Websocket RPC call."""
         call = RPCCall(self._next_id, method, params, self._route)
```

Comment:
Let's make sure we raise an error since this is called from outside of the class

```suggestion
        if self._client is None:
            raise RuntimeError("Not connected")

        call = RPCCall(self._next_id, method, params, self._route)
```